docs/data_pipeline_procedure.md

```diff
@@ -12,6 +12,45 @@
       find /global/cfs/cdirs/kbase/collections/$working_dir -type d -exec chmod g+s {} \;
       ```
 
+## Setting up Python environment using Pipenv
+   * Verify Python 3.11 installation
+     * To confirm Python 3.11 installation, run:
+       ```commandline
+       python --version
+       ```
+     * If needed, explore various methods to configure Python environments at NERSC using the [NERSC Python documentation](https://docs.nersc.gov/development/languages/python/nersc-python/)
+     * Example Usage with Conda
+       ```commandline
+       module load conda
+       conda create -n py311 python=3.11 pip
+       conda activate py311
+       ```
+   * Install [Pipenv](https://pipenv.pypa.io/en/latest/)
+     ```commandline
+     pip install pipenv
+     ```
+   * Pull Latest Code from GitHub
+     ```commandline
+     cd /global/cfs/cdirs/kbase/collections/collections
+     git pull origin main
+     ```
+   * Set up the Environment with Pipenv
+      ```commandline
+      pipenv install --dev --ignore-pipfile
```

Comment:
Shouldn't this be sync?

Reply:
Why ignore pipfile?

Reply:
since we checked in both pipfile and pipfile.lock, either way will work. I don't use pipenv sync personally. I just use the `--ignore-pipfile` flag with pipenv install. But I think either way it will work.

Reply:
Sync guarantees the same versions of dependencies as in the lock file, install doesn't, which is we we should be using sync for reproducible builds. IIUC canyway

Reply:
👍 updated the doc to use sync. I thought pipenv install --ignore-pipfile does the same thing. But yea. pipenv sync is forceful.

Reply:
:+1: